### PR TITLE
Make disambiguated org required

### PR DESCRIPTION
**Pull request by bobcaprice**

### CODE REVIEW

**Comment by bobcaprice** [comment]
@rcpeters @wjrsimpson @amontenegro 

Note: this PR has a schema change for removing and adding a unique constraint on the org table in a single transaction:

Liquibase:

<changeSet id="change_org_unique_constraint" author="George Nash" dbms="postgresql">
		<sql>ALTER TABLE org DROP CONSTRAINT org_unique_constraints;</sql>
		<sql>ALTER TABLE org ADD CONSTRAINT org_unique_constraints UNIQUE (name,city,region,country,org_disambiguated_id);</sql>
	</changeSet>

Tested on a copy of prod db on temp-db-2:

orcid=# begin                                                
;
BEGIN
Time: 0.155 ms
orcid=# ALTER TABLE org DROP CONSTRAINT org_unique_constraints;
ALTER TABLE
Time: 0.616 ms
orcid=# ALTER TABLE org ADD CONSTRAINT org_unique_constraints UNIQUE (name,city,region,country,org_disambiguated_id);
ALTER TABLE
Time: 3977.219 ms
orcid=# commit;
COMMIT
Time: 123.998 ms

**Comment by bobcaprice** [comment]
[3.0_dev1 for API make disambiguated organization required](https://trello.com/c/VduVk6oV/3994-30dev1-for-api-make-disambiguated-organization-required)

File: orcid-persistence/src/main/resources/db/updates/org_unique_constraints_add_disambiguated_org.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    =============================================================================
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">	
+	            
+	<changeSet id="change_org_unique_constraint" author="George Nash" dbms="postgresql">
+		<sql>ALTER TABLE org DROP CONSTRAINT org_unique_constraints;</sql>
+		<sql>ALTER TABLE org ADD CONSTRAINT org_unique_constraints UNIQUE (name,city,region,country,org_disambiguated_id);</sql>
```
Comment (amontenegro) [inline orcid-persistence/src/main/resources/db/updates/org_unique_constraints_add_disambiguated_org.xml]: Hey @bobcaprice,

Why are you adding the **org_disambiguated_id**  to the unique constraint?

Thanks

Comment (amontenegro) [reply orcid-persistence/src/main/resources/db/updates/org_unique_constraints_add_disambiguated_org.xml]: I talked with @rcpeters about it, it is good to go!

**Pull request merged**